src/xdocs/config_filters.xml

```diff
@@ -2232,6 +2232,285 @@ public class UserService {
       </subsection>
     </section>
 
+    <section name="SuppressWithNearbyTextFilter">
+      <p>Since Checkstyle 10.3.4</p>
```

Comment:
Since we had a release, this version and others in this PR needs to be updated.

Reply:
Done.

Reply:
This needs to be updated again.

Reply:
Leaving this for the end.